content/git/concepts/cherry-pick/cherry-pick.md

```diff
@@ -0,0 +1,79 @@
+---
+Title: 'cherry-pick'
+Description: 'The cherry pick command is used to select and put individual commits from one branch to another.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Developer Tools'
+Tags:
+  - 'Git'
+  - 'GitHub'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'learn-git'
+---
+
+The **cherry-pick** command entails picking individual [commits](https://www.codecademy.com/resources/docs/git/commit) from one [branch](https://www.codecademy.com/resources/docs/git/branch) to another.
+
```

Comment:
Add a syntax section showing the command by itself `git cherry-pick <commit>`. You can add information about some of the options available (e.g., -edit, -x) in the syntax section.